### PR TITLE
schema for SEG partitions

### PR DESCRIPTION
**Pull request by GiovanniVolta**
New schema to handle different definitions of AB and CD partition for SEG

### CODE REVIEW

File: xedocs/schemas/corrections/implementations/se_gain_partition.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""
+Correction: Region linear and circular for AB/Cd partitions
```
Comment (flammhead) [inline xedocs/schemas/corrections/implementations/se_gain_partition.py:2]:
```suggestion
Correction: Region linear and circular for AB/CD partitions
```

Sorry for being pedantic... 🙈

**Comment by LuisSanchez25** [comment]
The current schema already handels ab/cd partitions we dont need a new schema for this

**Comment by GiovanniVolta** [comment]
No, we need it since this defines what is AB and what is CD. Why are you saying that it is not needed?

**Comment by flammhead** [comment]
I quickly discussed with @GiovanniVolta about the additional Schema for partitioning and he agreed to try to incorporate the partitioning into the SEG schema. Though this likely is the more aesthetic solution, we should finish this PR as soon as reasonably possible, in order to not hold back the v15 reprocessing. So in case Giovanni does not manage to find a working solution by the end of his day, would @LuisSanchez25 be ok also with the present implementation? Thanks everyone!

**Comment by GiovanniVolta** [comment]
Working on correction [seg_partition](https://github.com/XENONnT/corrections/tree/seg_partition)

immagine

**Review by LuisSanchez25 — COMMENTED** [review]
2 quick comments, 
Since PARTITIONS are no longer used you can get rid of line 14.
Also as you mentioned these partitions will not just affect the single electron gain so maybe we should name it detector_partitions or something along those lines?

**Comment by GiovanniVolta** [comment]
> Since PARTITIONS are no longer used you can get rid of line 14.

Definitely! It's done

> Also as you mentioned these partitions will not just affect the single electron gain so maybe we should name it detector_partitions or something along those lines?

I think the name is okay since it affects all the SEG-related things such as avg_se_gain, se_gain a,d extraction eff (which is linked to se_gain).
I have updated the description in the schema to avoid misunderstanding in the future. Do you think it is enough?

**Comment by flammhead** [comment]
Hi @LuisSanchez25, thanks for looking into this PR! I would say, keeping the current naming is ok for this. I doubt that we are going to use these partitions for anything other than SE gain related in the future. Thanks everyone and have a nice weekend! :)

**Review by LuisSanchez25 — APPROVED** [review]

**Pull request merged**